ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Config.scala

```diff
@@ -27,6 +27,7 @@ final case class Config(
     shuffleParticipants: Boolean,
     partyAllocation: PartyAllocationConfiguration,
     ledgerClockGranularityMs: Int,
+    skipDarUpload: Boolean,
```

Comment:
I'd push this one further and change this to `uploadDars` too, with a default of `true`.

Reply:
fixed